### ISSUE TITLE
How should the parameters of the training kitti dataset be set

### ISSUE THREAD

**Issue by k794102577** [haoyu94/RoITr#12]
How should the parameters of the training kitti dataset be set

**Comment by Ruye-aa** [comment]
Hello, I have also been trying to conduct experiments on the KITTI dataset using RoITr recently, and found that the results are very poor due to issues with the computation of the normal vectors, which may be unrelated to the computation parameters

**Comment by k794102577** [comment]
> Hello, I have also been trying to conduct experiments on the KITTI dataset using RoITr recently, and found that the results are very poor due to issues with the computation of the normal vectors, which may be unrelated to the computation parameters

Hello, have you settled this question now

**Comment by CorgiBoyG** [comment]
> > Hello, I have also been trying to conduct experiments on the KITTI dataset using RoITr recently, and found that the results are very poor due to issues with the computation of the normal vectors, which may be unrelated to the computation parameters
> 
> Hello, have you settled this question now

Yes, I recently tried to experiment the model on modelnet40 as well as kitti and found that I ended up with poor results with the original parameters not adjusted. For example, on modelnet40, I calculated the error between the estimated transform matrix and the real transform matrix, e.g. **_rre is more than 40_**. How can I mitigate this problem and can you provide me with some good suggestions? Looking forward to your reply, thanks!

**Comment by allpassingby** [comment]
> > > 你好，我最近也在尝试使用 RoITr 在 KITTI 数据集上进行实验，发现结果很差，这是因为法向量的计算存在问题，这可能与计算参数无关
> > 
> > 
> > 你好，你现在解决这个问题了吗？
> 
> 是的，我最近尝试在 modelnet40 和 kitti 上试验该模型，发现由于原始参数未调整，最终结果不佳。例如，在 modelnet40 上，我计算了估计的变换矩阵与实际变换矩阵之间的误差，例如**_rre 大于 40_**。我该如何缓解这个问题，您能给我一些好的建议吗？期待您的回复，谢谢！

我在加载KITTI数据集报错了float和double问题，你有遇到过吗？
